apps/cde-ui/src/app/pages/visualization/visualization.component.scss

```diff
@@ -1,3 +1,6 @@
 :host {
   display: block;
+  min-width: 1280px;
```

Comment:
Why?

Reply:
According to the spec any size smaller than 1280px will use a scroll bar instead